### PR TITLE
Locale/nb-NO

### PR DESCRIPTION
**Pull request by santi**
Adds i18n support for Norwegian Bokmål, one of the official languages in Norway. Also updates docs to reflect the new language support.

### CODE REVIEW

File: components/locale-provider/nb_NO.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import Calendar from '../calendar/locale/nb_NO';
+
+export default {
+  locale: 'nb',
+  DatePicker,
+  TimePicker,
+  Calendar,
+  // Replace with rc-pagination/lib/locale/nb_NO
+  // when rc-pagination supports nb_NO
+  Pagination: {
```
Comment (benjycui) [inline components/locale-provider/nb_NO.tsx]: You  can PR to rc-pagination first, we can wait, thanks 😄

Comment (santi) [inline components/locale-provider/nb_NO.tsx]: Done. [Link to PR](https://github.com/react-component/pagination/pull/103). As soon as the Pagination's PR is merged I will update this PR.

**Comment by benjycui** [comment]
rc-pagination@1.10.5 had been published.

**Comment by afc163** [comment]
Thx~

**Pull request merged**